roles/configure_ztp_gitops_apps/tasks/main.yaml

```diff
@@ -103,30 +124,40 @@
         - "  - group: hive.openshift.io"
 
     - name: Add ClusterImageSet to the policies-app-project.yaml
-      lineinfile:
+      ansible.builtin.lineinfile:
         path: "{{ temp_dir.path }}/ztp/argocd/deployment/policies-app-project.yaml"
         line: "{{ item }}"
         insertafter: clusterResourceWhitelist.*
       loop:
         - "    kind: ClusterImageSet"
         - "  - group: hive.openshift.io"
 
-    - name: Run the policies and cluster apps
-      shell: |
-        {{ czga_oc_tool_path }} --kubeconfig="{{ temp_dir.path }}/kubeconfig" apply -k "{{ temp_dir.path }}/ztp/argocd/deployment"
-
     - name: Replace image container URL in argocd-openshift-gitops-patch.json
-      replace:
+      ansible.builtin.replace:
         path: "{{ temp_dir.path }}/ztp/argocd/deployment/argocd-openshift-gitops-patch.json"
         regexp: "{{ ztp_site_generator_image }}.*/"
         replace: "{{ ztp_site_generator_image }}:{{ ztp_site_generator_version }}"
 
     - name: Apply ArgoCD changes
-      shell: |
+      ansible.builtin.shell: |
         {{ czga_oc_tool_path }} --kubeconfig="{{ temp_dir.path }}/kubeconfig" patch argocd openshift-gitops -n openshift-gitops --patch-file "{{ temp_dir.path }}/ztp/argocd/deployment/argocd-openshift-gitops-patch.json" --type=merge
 
+    - name: Disable proxy addon
+      kubernetes.core.k8s:
+        api_version: multicluster.openshift.io/v1
+        kubeconfig: "{{ ztphub_kubeconfig_file | default(omit) }}"
```

Comment:
Where is `ztphub_kubeconfig_file` defined? Shouldn't you use `"{{ temp_dir.path }}/kubeconfig"` instead, similarly to what's done in the patch of the ArgoCD instance (the previous task)?

Reply:
This comes from a copy and paste of the task. The kubeconfig variable to be used should be czga_kubeconfig_path.
Updated.